src/___new___/data/featureTableComparisonInfo.ts

```diff
@@ -0,0 +1,349 @@
+          CalicoOpenSource: 'Y',
+          CalicoEntreprise: 'Y',
+          CalicoCloud: 'Y',
+        },
+      ],
+    },
+    {
+      title: 'Data',
+      content: [
+        {
+          rowHeader: 'Data-in-transit encryption for pod traffic using WireGuard',
+          CalicoOpenSource: 'Y',
+          CalicoEntreprise: 'Y',
+          CalicoCloud: 'Y',
+        },
+        {
+          rowHeader: 'SIEM integration',
+          CalicoOpenSource: '',
+          CalicoEntreprise: 'Y',
+          CalicoCloud: 'Y',
+        },
+      ],
+    },
+    {
+      title: 'Non-cluster hosts',
+      content: [
+        {
+          rowHeader: 'Restrict traffic to/from hosts using network policy',
+          CalicoOpenSource: 'Y',
+          CalicoEntreprise: 'Y',
+          CalicoCloud: 'N',
+        },
+        {
+          rowHeader: 'Automatic host endpoints',
+          CalicoOpenSource: 'Y',
+          CalicoEntreprise: 'Y',
+          CalicoCloud: 'N',
+        },
+        {
+          rowHeader: 'Secure Kubernetes nodes with host endpoints managed by Calico',
+          CalicoOpenSource: 'Y',
+          CalicoEntreprise: 'Y',
+          CalicoCloud: 'Y',
+        },
+        {
+          rowHeader: 'Apply policy to host-forwarded traffic ',
+          CalicoOpenSource: 'Y',
+          CalicoEntreprise: 'Y',
+          CalicoCloud: 'Y',
+        },
+      ],
+    },
+    {
+      title: 'Dataplane',
+      content: [
+        {
+          rowHeader: 'eBPF',
+          CalicoOpenSource: 'Y',
+          CalicoEntreprise: 'Y',
+          CalicoCloud: 'Y',
+        },
+        {
+          rowHeader: 'iptables',
+          CalicoOpenSource: 'Y',
+          CalicoEntreprise: 'Y',
+          CalicoCloud: 'Y',
+        },
+        {
+          rowHeader: 'Windows HNS',
+          CalicoOpenSource: 'Y',
+          CalicoEntreprise: 'Y',
+          CalicoCloud: 'Y',
+        },
+        {
+          rowHeader: 'VPP',
+          CalicoOpenSource: 'Y',
+          CalicoEntreprise: '',
+          CalicoCloud: '',
+        },
+      ],
+    },
+    {
+      title: 'Images',
+      content: [
+        {
+          rowHeader: 'Scan images for vulnerabilities',
+          CalicoOpenSource: '',
+          CalicoEntreprise: '',
+          CalicoCloud: 'Y',
+        },
+        {
+          rowHeader: 'Create policy to block vulnerable images from your clusters',
+          CalicoOpenSource: '',
+          CalicoEntreprise: '',
+          CalicoCloud: 'Y',
+        },
+      ],
+    },
+    {
+      title: 'Observability and troubleshooting',
+      content: [
+        {
+          rowHeader: 'Application level observability and troubleshooting',
+          CalicoOpenSource: '',
+          CalicoEntreprise: 'Y',
+          CalicoCloud: 'Y',
+        },
+        {
+          rowHeader: 'Service Graph',
+          CalicoOpenSource: '',
+          CalicoEntreprise: 'Y',
+          CalicoCloud: 'Y',
+        },
+        {
+          rowHeader: 'Threat graph',
```

Comment:
Do we have a threat graph? I'm not sure where I got this item. Is it for container threat defense?